pyfr/plugins/integrate.py

```diff
@@ -106,6 +106,10 @@ def _init_gradients(self, intg, rinfo):
         for ex in self.exprs:
             gradpnames.update(re.findall(r'\bgrad_(.+?)_[xyz]\b', ex))
 
+        privarmap = self.elementscls.privarmap[self.ndims]
+        self._gradpinfo = [(pname, privarmap.index(pname)) 
+                            for pname in self._gradpnames]
```

Comment:
Can use gradpnames directly here (and we no longer need to store self._gradpnames, since all of the relevant info is now in self._gradpinfo).

Reply:
Done